### ISSUE TITLE
#40 번역 중 Loss function과 관련된 번역 관련 조언부탁드립니다. 

### ISSUE THREAD

**Issue by insurgent92** [KerasKorea/KEKOxTutorial#47]
안녕하세요 #40 을 작업중에 있습니다. 매끄러운 번역을 하고싶은데 쉬운일이 아닙니다 :( 

> A loss function: the is how the network will be able to measure how good a job it is doing on its training data, and thus how it will be able to steer itself in the right direction.

> 손실 함수 : 손실함수는 학습 세트에 대해서 네트워크가 얼마나 잘 학습되었는지, 올바른 방향으로 학습되었는지에 대한 지표입니다.
우선은 이렇게 번역 했습니다. 오역/의견 피드백 부탁드립니다! 감사합니다 :D

**Comment by mike2ox** [comment]
@insurgent92 the network에 대한 부연설명을 넣으면 어떻까요
`(설계한) 네트워크가 주어진 학습 데이터에 잘 적합한 형태인지를 측정, 그로인해 어떻게 (네트워크 설계)방향을 조절할 것인지에 대한 지표`

**Comment by MijeongJeon** [comment]
꺄! 이런 이슈 너무 좋네요! 멋져요!

사실 기술 문서를 읽는 사람이 보기편하게 번역하는 일이란 여간 어려운 일이 아니랍니다.
두 분의 번역 모두 훌륭하네요~ 저도 번역해볼게요. 

> 손실 함수란 네트워크(혹은 모델)이 주어진 학습 데이터를 얼마나 잘 학습했는지 나타내는 함수입니다. 학습이 올바른 방향으로 진행 되고있는지 판단하는 지표로 주로 활용합니다.

고민에 참고해 주셔요~

**Comment by insurgent92** [comment]
@mike2ox @MijeongJeon 조언 감사합니다!
    
"* 손실 함수 : 손실 함수란 모델이 주어진 학습 데이터를 얼마나 잘 학습했는지 나타내는 함수입니다. 학습이 올바른 방향으로 진행 되고있는지 판단하는 지표로 주로 활용합니다.\n",

**Comment by RyanKor** [comment]
@insurgent92 님 안녕하세요. 함께 문서 작업을 하는 김승태 입니다.

'테스트 정확도는 97.8%를 기록했습니다 -- 이 결과는 사실 학습 세트의 정확도보다는 조금 낮습니다. 학습 정확도보다 테스트 정확도가 낮은 이유는 "과적합(overfilling)"과 관련이 있습니다. 기계학습 모델들은 학습 데이터에서의 성능보다 새로운 데이터에서 성능이 더 나빠지는 경향이 있습니다. 과적합 문제는 3장에서 자세히 다루도록 하겠습니다.'

작성하신 2.1 마지막 문단에 보시면 과적합(overfilling)이라 되어있는데요, 영단어를 overfitting으로 수정해주시면 감사하겠습니다!

**Comment by insurgent92** [comment]
@RyanKor 피드백 감사합니다!!
